Raven.Studio.Html5/App/models/database/documents/document.ts

```diff
@@ -35,7 +35,14 @@ class document implements documentBase {
 
         return propertyNames;
     }
-
+    getDocumentCollectionLabel(queryResult: any): string {
```

Comment:
Code duplication. 

I think you should have single function that based on global config + ravenEntityName or document returns its label.